hack/e2e-test.sh

```diff
@@ -208,13 +213,26 @@ echo ""
 echo "Events in cluster namespace"
 oc get events -n ${CLUSTER_NAMESPACE}
 
-echo "Waiting for install job to start and complete"
-
+echo "Waiting for the ClusterDeployment ${CLUSTER_NAME} to install"
 INSTALL_RESULT=""
-if go run "${SRC_ROOT}/contrib/cmd/waitforjob/main.go" "${CLUSTER_NAME}" "install"
+
+i=1
+while [ $i -le ${max_cluster_deployment_status_checks} ]; do
+  IS_CLUSTER_DEPLOYMENT_INSTALLED=$(oc get cd ${CLUSTER_NAME} | awk 'NR == 2 {print $4}')
```

Comment:
I would rather we use `jq` instead of `awk`. We do not consider the printer columns to be a hard API.